pal/src/misc/bstr.cpp

```diff
@@ -130,16 +130,20 @@ STDAPI_(BSTR) SysAllocStringLen(const OLECHAR *psz, UINT len)
  *  return value = void
  *
  ***********************************************************************/
-STDAPI_(void) SysFreeString(const OLECHAR* psz)
+STDAPI_(void) SysFreeString(BSTR bstr)
```

Comment:
> Preserve const?

The original API signature on Windows and also https://github.com/Microsoft/ChakraCore/blob/linux/pal/inc/rt/palrt.h#L444 does not have const.